common/src/main/java/org/conscrypt/NativeCrypto.java

```diff
@@ -46,12 +46,8 @@
 
 /**
  * Provides the Java side of our JNI glue for OpenSSL.
- *
- * @hide
  */
-@Internal
-public final class NativeCrypto {
-
+final class NativeCrypto {
```

Comment:
Just for the `X509_NAME_hash_old` method? Or do I need to make other methods public as well?